tkp/distribute/multiproc/tasks.py

```diff
@@ -26,6 +32,19 @@ def quality_reject_check(zipped):
 
 def extract_sources(zipped):
     logger.info("running extracted sources task")
-    url, args = zipped
+    accessor, args = zipped
     extraction_params = args[0]
-    return tkp.steps.source_extraction.extract_sources(url, extraction_params)
+    return tkp.steps.source_extraction.extract_sources(accessor,
+                                                       extraction_params)
+
+
+def get_accessors(zipped):
+    logger.debug("Creating accessors for images")
+    images, args = zipped
+    return tkp.steps.persistence.get_accessors(images)
+
+
+def get_start_time(zipped):
+    logger.debug("Retrieving start time from accessors")
+    images, args = zipped
+    return [a.taustart_ts for a in tkp.steps.persistence.get_accessors(images)]
```

Comment:
well, images is just a list with one element here. This is a bit weird but that is how the mapping stuff works in trap, it supports supplying a list of things per map function.

Also, the image gets out of scope so it should be cleaned up by the garbage collector.

Lastly this function is called by the mapper in parallel using multiproc, and by default it would spawn a number of processes equal to the number of cores. The returned value is just a list of timestamps.